### PR TITLE
Aligned .close() method with the standard, allowing you to pass code …

### PR DESCRIPTION
**Pull request by mchaov**
Aligned .close() method with the standard, allowing you to pass code and reason for closing.

### CODE REVIEW

**Comment by mchaov** [comment]
Great job on the wrapper :) You saved me quite some time mate.

I can publish this in a private repository so I am not going to pester you to merge and release a new package. However, it seems the ability to add code and reason for closing is going to be useful for others as well.

There are some other inconsistencies with the standard, if I get the time to patch them will send you another PR.

File: src/index.js [hunk 1]
```diff
@@ -255,10 +255,10 @@ class WebSocketAsPromised {
    *
    * @returns {Promise<Event>}
    */
-  close() {
+  close(code, reason) { // https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close
```
Comment (vitalets) [inline src/index.js]: Could you add jsdoc for params.

File: src/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Channel = require('chnl');
 const PromiseController = require('promise-controller');
 const Requests = require('./requests');
 const defaultOptions = require('./options');
-const {throwIf} = require('./utils');
+const { throwIf } = require('./utils');
```
Comment (vitalets) [inline src/index.js]: Could you please keep codestyle.

Comment (mchaov) [reply src/index.js]: sorry, didn't notice this one - it is auto from the rules we apply internally :)

File: src/index.js (hunk 1 above)
Comment (mchaov) [reply src/index.js]: kk

**Comment by mchaov** [comment]
:) review please

File: src/index.js [hunk 3]
```diff
@@ -303,10 +305,10 @@ class WebSocketAsPromised {
   _createWS() {
     this._ws = this._options.createWebSocket(this._url);
     this._wsSubscription = new Channel.Subscription([
-      {channel: this._ws, event: 'open', listener: e => this._handleOpen(e)},
-      {channel: this._ws, event: 'message', listener: e => this._handleMessage(e)},
-      {channel: this._ws, event: 'error', listener: e => this._handleError(e)},
-      {channel: this._ws, event: 'close', listener: e => this._handleClose(e)},
+      { channel: this._ws, event: 'open', listener: e => this._handleOpen(e) },
```
Comment (vitalets) [inline src/index.js]: Please, keep codestyle also here and below.

**Pull request merged**